src/3d/shaders/pointcloud.frag

```diff
@@ -125,4 +131,14 @@ void main(void)
     color = vec4(pointColor, 1.0f);
     break;
   }
+
+  //Apply light
+  if (triangulate)
+  {
+      float ambianceFactor=0.15;
```

Comment:
what is this factor about?

Reply:
well, here, I keep the same code used for 3D mesh, and honestly I don't remember well... I think, this value is necessary to avoid too bright triangles. I chose it after several tries, with the one that give the better result with a reasonable light intensity.

Reply:
Okay... it is probably worth commenting it in the code that this value was picked empirically by visually checking results (rather than being derived from a formula)